iTerm2XCTests/VT100GridTest.m

```diff
@@ -53,6 +53,9 @@ - (screen_char_t)gridBackgroundColorCode {
 - (void)gridCursorDidChangeLine {
 }
 
+- (void)gridCursorDidChangeCol {
```

Comment:
Abbreviations should never be used in new code (there's some old crusty code that hasn't been cleaned up yet). Please change all shortenings of "vertical", "column", "cursor", etc into the full word.

Reply:
Done.